packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1086,6 +1086,8 @@ function renderRoot(root: FiberRoot, isYieldy: boolean): void {
     if (__DEV__) {
       resetStackAfterFatalErrorInDev();
     }
+    // Clear the currently rendering root.
```

Comment:
These comments are not helpful. Obviously you're clearing the root based on the next line. Say *why*, not *what* you're doing.

Reply:
E.g. "By clearing the root, we force React to start over by rendering the root from scratch instead of continuing in the middle of the tree. We have to do that because we've added new work to the tree and that might otherwise be inconsistent depending on where we are in the current render."

Reply:
Ok Seb, I know what a comment is. Thanks for the explanation.

Reply:
"Better comment, please" would have sufficed